app/src/main/java/com/kickstarter/viewmodels/projectpage/AddOnsViewModel.kt

```diff
@@ -150,7 +150,6 @@ class AddOnsViewModel(val environment: Environment, bundle: Bundle? = null) : Vi
                         bonusAmount = b.amount()
                     } else {
                         backedAddOns = b.addOns() ?: emptyList()
-                        currentUserReward = b.reward() ?: currentUserReward
```

Comment:
this was causing the issue mentioned [here](https://github.com/kickstarter/android-oss/pull/2173/files#r1848962215)